app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/model/Dataset.kt

```diff
@@ -1,8 +1,28 @@
 package com.github.HumanLearning2021.HumanLearningApp.model
 
-import java.io.Serializable
+import android.os.Parcelable
 
-interface Dataset: Serializable {
+/**
+ * id should be used to uniquely identify the Dataset
+ */
```

Comment:
useful comment, but misplaced, no?

Reply:
I would argue no because the design of the other interfaces depends on it

Reply:
No I mean it should be just above the attribute (look at the file for context)

Reply:
Oh ok, I see what you mean.